Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,79 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
+This is the default end-of-checkout script.
+NOTE: Prepare to take fltas only if you have sufficient time to vent the telescope afterwards.
+
+It will leave the telescope in position to start taking afternoon calibraitons. This is az = 205.7 elev = 30.
+If you are not sure how you want to leave the telescope, consider running this script as it will perform a few more useful checks of the system.
```

Comment:
I don't quite understand this.  This script puts the telescope in a specific position, so why run it if you are not sure where you want the telescope?
Are you trying to say that if you want the telescope in a different position than for the flats, then it is still useful to use this script because it opens everything for you?

Reply:
I took it from the official web page to make consistent with what is there, but I guess you are right, is kinda confusing...but yes I guess it is regarded to fact that if you do not have time to run the calibration you can run the script to leave the AT in that position and continue with the next steps of the startup.